README.md

```diff
@@ -2,6 +2,67 @@
 
 These are the scripts I use while releasing [The Foreman](https://theforeman.org). They automate most of the [release process](https://github.com/theforeman/tool_belt/blob/master/procedures/foreman/release.md.erb).
 
+## Setup
+
+Required dependencies:
+
+ * gopass
+ * python3-devel
+   * click
+   * python-jenkins
+
+### Gopass
+
+First, clone this repository to your local machine. [gopass](https://github.com/gopasspw/gopass) will need to be installed and then set up on your machine for the release environment.
+
+Ensure that `gopass` is initialized after installing the first time (and that your GPG private key is present on the system):
+
+```
+gopass init
+```
+
+After installing `gopass`, the shared password stores need to be cloned. This requires setting up SSH configuration to the Foreman puppetmaster:
+
+Add to ~/.ssh/config:
+
+```
+Host foreman-pm
+  HostName puppetmaster.theforeman.org
+  Port 8122
+  User <your SSH user>
+  LocalForward 9443 localhost:443
+  ExitOnForwardFailure yes
+```
+
+Now clone:
+
+```
+gopass clone --sync gitcli foreman-pm:/srv/secretsgit/theforeman-release.git theforeman/releases
+gopass clone --sync gitcli foreman-pm:/srv/secretsgit/theforeman-passwords.git theforeman/shared
+```
+
+You will need to create a few gopass passwords for jobs:
+
+For commands on the puppetmaster:
```

Comment:
```suggestion
For commands on the puppetmaster, add your `sudo` password:
```

Reply:
We also don't perform any commands on the puppetmaster. This is only to store tarball signatures on web02.